Without this by default you see e.g. "OS: Chimera x86_64" instead of
"OS: Chimera Linux x86_64" similar to neofetch.

Instead of using NAME from /etc/os-release prefer PRETTY_NAME
effectively making --os-format '{3} {12}' the default.

--- a/src/modules/os/os.c
+++ b/src/modules/os/os.c
@@ -12,10 +12,10 @@
 static void buildOutputDefault(const FFOSResult* os, FFstrbuf* result)
 {
     //Create the basic output
-    if(os->name.length > 0)
-        ffStrbufAppend(result, &os->name);
-    else if(os->prettyName.length > 0)
+    if(os->prettyName.length > 0)
         ffStrbufAppend(result, &os->prettyName);
+    else if(os->name.length > 0)
+        ffStrbufAppend(result, &os->name);
     else if(os->id.length > 0)
         ffStrbufAppend(result, &os->id);
     else
